### DISCUSSION TITLE
Keycloak extension use service account

### DISCUSSION THREAD

**Discussion by bulldog98** [quarkusio/quarkus#32905] [Q&A] [answered]
Is there a way to use the extension to get a token to send to another service, when you start from a non user input?

[a comment by an automated account is not shown]

**Comment by sberyozkin** [comment, marked as answer]
@bulldog98 Please see a Keycloak specific option: https://quarkus.io/guides/security-keycloak-admin-client and more general OidcClient option: https://quarkus.io/guides/security-openid-connect-client-reference

**Reply by bulldog98** [reply]
Thanks for pointing me in the right direction, I got lost in the other guides.
